src/CreativeCollab.css

```diff
@@ -0,0 +1,16 @@
+.editor-container{
+    margin: 0 auto;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.story-container{
+    position: absolute;
+    border: 1px solid black;
+    padding-left: 1%;
+    bottom: 5%;
+    left: 20%;
+    width: 800px;
+    height:380px;
+}
```

Comment:
nit: new line at end of file pretty please!

Reply:
✅